DigitalLearningSolutions.Web/Scripts/nhsuk.ts

```diff
@@ -18,4 +19,5 @@ document.addEventListener('DOMContentLoaded',
     SkipLink();
     Radios();
     Checkboxes();
+    Card();

```

Comment:
How come we didn't need this before now? 🙂

Reply:
The only clickable card that we have is for the course summary in the initial menu page. We need JS for the card to behave like a link properly, but I didn't notice this because the card was only given a real link (instead of a placeholder of `href="#"`) as part of cd088ea 🙂